share/completions/iptables.fish

```diff
@@ -1,103 +1,104 @@
 set -l __fish_iptables_tables filter nat mangle raw security
 
 function __fish_iptables_current_table
-	set -l next_is_table 1
-	for token in (commandline -oc)
-		switch $token
-			case "--table=*"
-				set -l IFS "="
-				echo $token | while read a b
-					echo $b
-				end
-				return 0
-			case "--table"
-				set next_is_table 0
-			case "-*t*"
-				set next_is_table 0
-			case "*"
-				if [ $next_is_table -eq 0 ]
-					echo $token
-					return 0
-				end
-		end
-	end
-	return 1
+    set -l next_is_table 1
+    for token in (commandline -oc)
+        switch $token
+            case "--table=*"
+                set -l IFS "="
+                echo $token | while read a b
+                    echo $b
+                end
+                return 0
+            case "--table"
+                set next_is_table 0
+            case "-*t*"
+                set next_is_table 0
+            case "*"
+                if [ $next_is_table -eq 0 ]
+                    echo $token
+                    return 0
+                end
+        end
+    end
+    return 1
 end
 
 function __fish_iptables_user_chains
-	# There can be user-defined chains so we need iptables' help
-	set -l tablearg
-	set -l table (__fish_iptables_current_table)
-	if __fish_iptables_current_table
-		set tablearg "--table=$table"
-	end
-	# This only works as root, so ignore errors
-	iptables $tablearg -L ^/dev/null | grep Chain | while read a b c
-		echo $b
-	end
+    # There can be user-defined chains so we need iptables' help
+    set -l tablearg
+    set -l table (__fish_iptables_current_table)
+    if __fish_iptables_current_table
+        set tablearg "--table=$table"
+    end
+    # This only works as root, so ignore errors
+    iptables $tablearg -L ^ /dev/null | grep Chain | while read a b c
+        echo $b
+    end
 end
 
 
 function __fish_iptables_chains
-	set -l table (__fish_iptables_current_table)
-	[ -z $table ]; and set -l table "*"
```

Comment:
Yeah - how about only breaking to achieve line length? With the possible exception of "if; begin; a; and; b; end;" blocks.